### DISCUSSION TITLE
2023/04/21 Task

### DISCUSSION THREAD

**Discussion by JeongJaeSoon** [2P3S/scrum-dice#44] [General]
## 승현
* [x] Design 수정요소 Fix
* [x] #43 

## 재순
* https://github.com/2P3S/2p3s-backend-system/issues/1
	* [x] 채팅 방에 재접속
	* [x] 사용자 접속 유지

## 승형
* https://github.com/2P3S/2p3s-backend-system/issues/5
	* [ ] Socket 부분 설계
	    * https://miro.com/app/board/uXjVMSIDdFM=/
    	* 어느 정도 작성된 뒤에, 수시로 피드백 받기

**Comment by JeongJaeSoon** [comment]
# 승현
https://www.figma.com/file/rhYzh5jurgqhfMt1m6wDT3/Scrum-Dice?node-id=0-1&t=CFnlWmYBcOJpywNw-0

**Comment by Samsam-lee** [comment]
- 승현
    - 1안 2안 → 모든 카드가 보이도록 UI를 수정
        - 위 아래로 나뉘어져 있지만, 좌우로 변경하는 편이 좋아보임
    - 커스텀 카드 → 옵션 카드
    - url 복사 버튼 추가
    - 플래닝 포커 방 안에서 색이 많은 것 같다.
    - 플래닝 포커 기록에서 앞뒤 버튼 추가
    - 검색 창 부분은 아웃라인을 제거하고 배경색을 넣는걸로
    - 고려 할 항목
        - 시간을 표시하는 부분 크기?
        - 버튼 위치를 이동시켜 두 줄로 변경시키는 것이 어떤가?
        - 카드 아래에 영어로 이름을 제거?
    - 스토리북
        - 다른 사람이 봤을 때, 요소들을 볼 수 있음
        - props 같은 것들도 디테일하게 테스트 가능
- 재순
    - vscode에서 도큐먼트 확인 가능하게 만듬
    - redis 모듈 구현
        - 시간을 지정해주지 않으면 영구보존 (해결되지 않아서 99999초 이런식으로 지정)
        - 우리가 추구하는 방식과 맞지 않음
    - mongoDB 사용이유?
        - RDB를 사용할 때, 의존 관계도 설정해줘야하고, 테이블 별로 데이터를 나눠줘야하기 때문에
        - 멤버의 로그를 볼 때는 별도의 스키마를 짤 필요가 있다.
    - nestJS 개발을 해보니 기본 구조가 클린 아키텍쳐와 비슷하다!
